builds/jazz-build-module/sls-app/whitelist.yml

```diff
@@ -483,3 +483,8 @@ assetTypes:
   - AWS::Kinesis::Stream
   - AWS::SQS::Queue
   - AWS::Events::Rule
+
```

Comment:
I see that we are having to maintain this separately. In this current implementation, if I add additional fields or remove fields as mandatory=true, I will have to update this list. Can this be derived from serverless-build-rules.yml instead?

Reply:
Implemented the suggestion